src/analyses/base.ml

```diff
@@ -657,7 +660,6 @@ struct
               VD.top () (* upcasts not! *)
           in
           let v' = VD.cast t v in (* cast to the expected type (the abstract type might be something other than t since we don't change addresses upon casts!) *)
-          M.tracel "cast" "Ptr-Deref: cast %a to %a = %a!\n" VD.pretty v d_type t VD.pretty v';
```

Comment:
Why is this removed? If anything, it should actually have `if M.tracing then ` added to it.